website/content/partials/known-issues/transit-managed-keys-panics.mdx

```diff
@@ -0,0 +1,16 @@
+### Transit Encryption with Cloud KMS managed keys causes a panic
+
+#### Affected versions
+
+- 1.13.1+ up to 1.13.7 inclusively
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+Vault will panic when a Transit encryption API call is backed by a Cloud KMS managed key (Azure, GCP, AWS).
+
+<Note>Encryption and decryption APIs are not affected with PKCS#11 managed keys and non-managed key, key types</Note>
```

Comment:
```suggestion
<Note>Encryption and decryption APIs are not affected with PKCS#11 managed keys and transit native keys</Note>
```

Reply:
Thanks you beat me to updating from cli.

Reply:
Whoops :P